Server/Plugins/APIDump/Classes/Plugins.lua

```diff
@@ -5,6 +5,21 @@ return
 		Desc = "cPlugin describes a Lua plugin. Each plugin has its own cPlugin object.",
 		Functions =
 		{
+			AddWebTab =
```

Comment:
Wrong branch?

Reply:
Not really, I removed it at first because tolua wouldn't parse the `using` clause; by the time I found more of those, it was too late to go back. I don't want to change this PR any more, it's huge as it is already.

Reply:
>I don't want to change this PR any more, it's huge as it is already.

That's exactly why it shouldn't contain unrelated changes. Reviewing the diff would be a nightmare.